site/en/blog/cvd/index.md

```diff
@@ -0,0 +1,337 @@
+  <figcaption>
+    The impact of simulating tritanopia on a colorful picture of melted crayons.
+  </figcaption>
+</figure>
+
+As a developer with regular vision, you might see DevTools display a bad contrast ratio for color pairs that visually look okay to you. This happens because the contrast ratio formulas take into account these color vision deficiencies! _You_ might still be able to read low-contrast text in some cases, but people with vision impairments don't have that privilege.
+
+By letting designers and developers simulate the effect of these vision deficiencies on their own web apps, we aim to provide the missing piece: not only can DevTools help you *find* and *fix* contrast issues, now you can also *understand* them!
+
+## Simulating color vision deficiencies with HTML, CSS, SVG, and C++
+
+Before we dive into the Blink Renderer implementation of our feature, it helps to understand how you'd implement equivalent functionality using web technology.
+
+You can think of each of these color vision deficiency simulations as an overlay covering the entire page. The Web Platform has a way to do that: CSS filters! With the CSS `filter` property, you can use some predefined filter functions, such as `blur`, `contrast`, `grayscale`, `hue-rotate`, and many more. For even more control, the `filter` property also accepts a URL which can point to a custom SVG filter definition:
+
+```html
+<style>
+  :root {
+    filter: url(#deuteranopia);
+  }
+</style>
+<svg>
+  <filter id="deuteranopia">
+    <feColorMatrix values="0.367  0.861 -0.228  0.000  0.000
+                           0.280  0.673  0.047  0.000  0.000
+                          -0.012  0.043  0.969  0.000  0.000
+                           0.000  0.000  0.000  1.000  0.000">
+    </feColorMatrix>
+  </filter>
+</svg>
+```
+
+The above example uses a custom filter definition based on a color matrix. Conceptually, every pixel's `[Red, Green, Blue, Alpha]` color value is matrix-multiplied to create a new color `[R′, G′, B′, A′]`.
+
+Each row in the matrix contains 5 values: a multiplier for (from left to right) R, G, B, and A, as well as a fifth value for a constant shift value. There are 4 rows: the first row of the matrix is used to compute the new Red value, the second row Green, the third row Blue, and the last row Alpha.
+
+You might be wondering where the exact numbers in our example come from. What makes this color matrix a good approximation of deuteranopia? The answer is: science! The values are based on [a physiologically accurate color vision deficiency simulation model by Machado, Oliveira, and Fernandes](https://www.inf.ufrgs.br/~oliveira/pubs_files/CVD_Simulation/CVD_Simulation.html ).
+
+Anyway, we have this SVG filter, and we can now apply it to arbitrary elements on the page using CSS. We can repeat the same pattern for other vision deficiencies. Here's a demo of what that looks like:
+
+{% Glitch {
+  id: 'color-vision-deficiencies',
```

Comment:
Looks like the styles are missing from the Glitch. Could you please update it so it matches the original gist? https://gist.github.com/mathiasbynens/1ef1612251ebbb8c51f2e2c851a0c365#file-demo-html

Reply:
Added